libs/handoversidesheet/src/lib/ui-sidesheet/HandoverSidesheet.tsx

```diff
@@ -32,7 +38,13 @@ type HandoverProps = {
   item?: HandoverPackage;

```

Comment:
Look at Workorder sidesheet and use the SkeletonSidesheet from @cc-components/sharedcomponents